src/test/scenarios/datafactory/output/src/datafactory-preview/azext_datafactory_preview/tests/latest/test_datafactory_scenario.py

```diff
@@ -130,7 +206,17 @@ def test_datafactory_Scenario(self, rg):
             'myIntegrationRuntime2': 'exampleManagedIntegrationRuntime',
         })
 
+
+    @ResourceGroupPreparer(name_prefix='clitestdatafactory_exampleResourceGroup'[:7], key='rg', parameter_name='rg')
+    def test_datafactory_Scenario(self, rg):
         call_scenario(self, rg)
         calc_coverage(__file__)
         raise_if()
 
+
+    @ResourceGroupPreparer(name_prefix='clitestdatafactory_exampleResourceGroup'[:7], key='rg', parameter_name='rg')
+    def test_datafactory_Scenario_min(self, rg):
+        call_scenario_min(self, rg)
+        calc_coverage(__file__)
+        raise_if()
+
```

Comment:
If I have manually override the scenarios because the autogenerated order won't be able to work. Does this mean I have to override the minimal scenario as well ?

Reply:
yes, override need to be done in minimal scenario also in this case.

If normal test and minimal test share the same example order, it's recommended to define the order in test-scenario configuration